### ISSUE TITLE
"timer ended" message when Awake Time spans midnight

### ISSUE THREAD

**Issue by BStudent** [jonathanlermitage/intellij-night-and-day-plugin#3]
When used to countdown remaining time in day and "Awake Mode" hours are set to realistic programmers hours  :-)  such as 11:00 to 02:00, status bar shows "timer ended."
Mouse-over shows error message that "11:00 > 02:00".  
Could be good to use (what in python is) <start_time> + <24 hour datediff>.

**Comment by jonathanlermitage** [comment]
@BStudent Yeah, the calculation of the remaining time in awake mode was a nightmare for me 😁  
Adding support for workdays across 2 days would be a great addition. I don't have the motivation to implement this feature, but contributions are highly appreciated, and I can provide some assistance if needed. Just for info, I think we would need to improve the [`bewteen` method](https://github.com/jonathanlermitage/intellij-night-and-day-plugin/blob/main/src/main/java/lermitage/intellij/nightandday/core/DateUtils.java#L186), and rework the GUI.

**Comment by BStudent** [comment]
LoL I'll get back to you on that when I start programming in Java, which I've avoided for 20+ years.
But if you need advice on python HMU.
BTW, it appears our cats share an interest in fashion eyewear...

![image](https://user-images.githubusercontent.com/19792522/209486455-519391c0-4b95-4b90-821d-4a8b3cc8bf1c.jpeg) 
   

"Silent Judgment Cat looks at you and says nothing..." :-)
